Client/game_sa/CStreamingSA.cpp

```diff
@@ -152,3 +152,8 @@ void CStreamingSA::ReinitStreaming()
 
     reinitStreaming();
 }
+
+void CStreamingSA::MakeSpaceFor(std::uint32_t memoryToCleanInBytes)
+{
+    ((int(__cdecl*)(std::uint32_t))0x40E120)(memoryToCleanInBytes);
```

Comment:
```suggestion
    (reinterpret_cast<void(__cdecl*)(std::uint32_t)>(0x40E120))(memoryToCleanInBytes);
```
That function doesn't return anything.